exercises/secret-handshake/instructions.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+Your task is to converts a number between 1 and 31 to a sequence of actions in the secret handshake.
+
+The sequence of actions are chosen by looking at each digit of the number once it's been converted to binary.
```

Comment:
Maybe be explicit about which digits to check?

```suggestion
The sequence of actions are chosen by looking at the rightmost five digits of the number once it's been converted to binary.
```

Reply:
The largest number in the range is 5 digits long in binary, so we never have more digits than that.

Reply:
True, but I could imagine programmers thinking about numbers in terms of bytes and multiples of 8 bits.

Reply:
Yes, that's a good point.

Reply:
I'm going to add this, and then maybe someone in the problem-specs repo will have some other good idea.